backend/src/main/java/hu/flowacademy/meteo/config/SwaggerConfig.java

```diff
@@ -0,0 +1,22 @@
+package hu.flowacademy.meteo.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
+    @Bean
+    public Docket productApi() {
+        return new Docket(DocumentationType.SWAGGER_2)
```

Comment:
Szépen mutatja az api-t, érdemes lenne kipróbálni az elérhető controllereket és logot és error handlinget is adni hozzá, mert most még nincs (ha jól látom).